lifechanger/src/app/experience-info/experience-info.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="experience-container">
+  <div class="h4-container">
+    <h4>{{ experienceInfo.title }}</h4>
+    <button class="btn-work">{{ experienceInfo.workType }}</button>
+  </div>
+  <div class="work-container">
+    <div class="div1-experience">
+      <div class="div2-experience">
+        <p class="p-work">
+          <img src="../../assets/experience-info/grid.svg" />
+          {{ experienceInfo.project }}
+        </p>
+        <p class="p-work">
+          <img src="../../assets/experience-info/location.svg" />
+          {{ experienceInfo.location }}
+        </p>
+      </div>
+      <p class="p-work">
+        <img src="../../assets/experience-info/calendar.svg" />
+        {{ experienceInfo.startDate | date }} -
+        {{ experienceInfo.endDate | date }}
+      </p>
+    </div>
+  </div>
+</div>
+<div class="line"></div>
```

Comment:
check if measurements can be switched to dynamic (or just remove it),
+BEM re-naming